boards/tsi/ek_tsi_skyp/board.cmake

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2024 TSI
+# SPDX-License-Identifier: Apache-2.0
+
+
+set(SUPPORTED_EMU_PLATFORMS qemu armfvp)
+
+# QEMU settings
+set(QEMU_CPU_TYPE_${ARCH} cortex-m55)
+set(QEMU_FLAGS_${ARCH}
+  -cpu ${QEMU_CPU_TYPE_${ARCH}}
+  -machine mps3-an547
```

Comment:
Are you using mps3-an547 for QEMU ?

Reply:
We use **board.cmake** for hardware flashing (**west flash**). Currently, we are not using **board.cmake**.